### ISSUE TITLE
Chunkloading Issue when used with Ae2

### ISSUE THREAD

**Issue by Karechta** [RS485/LogisticsPipes#1566]
https://github.com/RS485/LogisticsPipes/blob/b7e45375a7c54d72db1dda478c108573c33320ac/common/logisticspipes/routing/pathfinder/changedetection/TEControl.java#L121

If i am not wrong, "blockExists" loads a Chunk if needed, because of which Blockupdates of Blocks cause the Chunk to load, this is an Issue if used with AE2, since BaseTiles of Ae2 are being BLockupdated pretty often (not sure when exactly thats happening) and it happenes consistantly that those Tiles even get an BlockUpdate although the Chunk is no longer loaded. 
At this point the TEControl.java takes those Blockupdates and to do what its supposed to do it loads in the Chunk, and I think it never unloads them. This way it just creates a Chunkloader recreatable with just 3 Blocks.
After Server Restarts the Chunks load in again, without players being near it.

Our testings are Based on the mod - Versions: 
Logistics Pipe: 0.10.3.73
Ae2: rv6-stable-7

There is also an Issue at Ae2 already:
https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3957
It says its closed since the Po3 "fixed it" but thats not the case, since i talked to them about it, and they dont know how to fix it. 
also on Po3:
https://github.com/Cazadorsniper/PO3/issues/188

AE2 "pushes" Blocks into blockupdater here: 
https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/rv6-1.12/src/main/java/appeng/tile/AEBaseTile.java#L311

A Stacktrace to how we found out about it:
https://imgur.com/a/KKo12um

TLDR: 
TEControl Takes Blockupdates and loads in the Chunks(if not already loaded) in which those happen.
Ae2 "Pushes" their Tiles into The Blockupdate regularly which causes the Chunk in which they are to load even if the chunk was previously unloaded and no player was around.

**Comment by bziemons** [comment]
Thanks for the detailed description. What would you suggest to change on our side?

**Comment by Karechta** [comment]
Best Thing would probably be to cooperate with the AE2 Dev, since both mods probably have their own use of certain handlers or events. Personally i dont have an Idea, since i didnt check how much your Mod would be affected by some changes.

**Comment by bziemons** [comment]
@yueh any idea why we are receiving block updates of non-loaded chunks?
